app/views/people/_person.html.haml

```diff
@@ -1,17 +1,18 @@
 .speaker
   %a{ id: person.slug }
   .speaker__headshot
-    - if person.image_filename
```

Comment:
Hi Kara, thanks for fixing the other two files, this file also needs the updated convention. :-)